qemu/tests/cfg/block_resize.cfg

```diff
@@ -59,3 +59,14 @@
         disk_change_ratio = 1.5 0.5
     RHEL.5:
         need_reboot = yes
+    variants:
+        - @default:
+        - data_plane:
+            only Host_RHEL.m7
+            only virtio_blk virtio_scsi
+            iothreads = iothread0
+            virtio_blk:
+                blk_extra_params_stg += ",iothread=${iothreads}"
+            virtio_scsi:
+                no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
+                bus_extra_params_stg = "iothread=${iothreads}"
```

Comment:
how to get drive_id for virtio_scsi as you didn't transfer wwn|serial

Reply:
for virtio-scsi, "iothread=${iothreads}" is added to virtio-scsi-pci, no effect to blk params.

Reply:
@suqinhuang  wwn/serial was defined in line 13 and 15 in block_resize.cfg